lib/editing/editor_codemirror.dart

```diff
@@ -179,6 +210,12 @@ class _CodeMirrorEditor extends Editor {
   set completionAutoInvoked(bool value)
       => cm.jsProxy['state']['completionAutoInvoked'] = value;
 
+  bool get lookingForQuickFix
```

Comment:
I've tried storing information in the codemirror editor as fields, or private fiels, but this informatotion get lost, as there is new instance created of the editor at the beginning of the completionhelper. Therefore I think we must store it to the jsobject instead.